specification/cognitiveservices/HealthInsights/healthinsights.openapi/tspconfig.yaml

```
@@ -6,5 +6,7 @@ emit: [
 
 options:
   "@azure-tools/typespec-autorest":
-     output-file: "openapi.json"
+     azure-resource-provider-folder: "HealthInsights"
+     emitter-output-dir: "{project-root}/../../data-plane"
+     output-file: "{azure-resource-provider-folder}/{version-status}/{version}/openapi.json"
```

```suggestion
     azure-resource-provider-folder: "./data-plane"
     emitter-output-dir: "{project-root}/../.."
     output-file: "{azure-resource-provider-folder}/{service-name}/{version-status}/{version}/openapi.json"
```

Would this be better?  I think this is equivalent and it aligns with `Contoso.WidgetManager` (assuming the sample is most correct).